src/hooks/useTracking.js

```diff
@@ -1,38 +1,21 @@
 import { useRef, useEffect } from 'preact/hooks'
-import { appVersion, sendEvent, isInstrumentationEnabled } from 'utils'
+import {
+  appVersion, appInstallId, generateRandomId, sendEvent,
+  isInstrumentationEnabled
+} from 'utils'
 
 const SCHEMA_NAME = 'InukaPageView'
 const SCHEMA_REV = 19883738
-const USER_ID_KEY = 'INUKA-PV-U'
 const SESSION_ID_KEY = 'INUKA-PV-S'
```

Comment:
just checking the latest schema, the `user_id` said "Randomly generated user id, expiring after 90 days of inactivity.", but the `appInstalledId` never expires once user starts using the app.

is that fine we don't set the time checking same as the session?

Reply:
This is generally true regardless of the implementation since all that data is deleted on the server after 90 days.